clusterloader/cluster_loader.go

```diff
@@ -156,32 +158,16 @@ func createTemplate(baseName string, ns *v1.Namespace, configPath string, numObj
 		if tuning != nil {
 			if tuning.Templates.RateLimit.Delay != 0 {
 				framework.Logf("Sleeping %d ms between template creation.", tuning.Templates.RateLimit.Delay)
-				time.Sleep(time.Duration(tuning.Templates.RateLimit.Delay) * time.Millisecond)
+				time.Sleep(tuning.Templates.RateLimit.Delay * time.Millisecond)
```

Comment:
I know - but if you are setting "duartion: 1" it is what you are asking for.
No matter what it is, if I'm passing "time.Duration" to a function, I'm expecting it to operation on a given duration not on duration multipled by something.